src/bull/crawler.service.spec.ts

```diff
@@ -18,12 +18,19 @@ describe('CrawlerService', () => {
 
   describe('getSearchResultData', () => {
     it('should return HTML content from Google search', async () => {
-      const keyword = 'testKeyword';
+      const mockResponse = 'Mocked Google Search Results';
```

Comment:
This mock is too different from an actual response. Thus, the test does not match how the module works, and that reduces the value of the test. This solution also bypasses the expectation of using a tool such as [PollyJS](https://netflix.github.io/pollyjs/#/) to mock HTTP request

Reply:
Yes, I agree with that. If I managed to finish other feedbacks, I will try to revisit it again